plugins/base/src/main/kotlin/utils/alphabeticalOrder.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.dokka.base.utils
+
+
+/**
+ * Canonical alphabetical order to sort named elements
+ */
+internal val canonicalAlphabeticalOrder: Comparator<String?> = nullsFirst(String.CASE_INSENSITIVE_ORDER).thenBy { it }
```

Comment:
or perhaps it should be `Comparator<String>` and whether to put nulls first or last should be decided on usage

Reply:
To me it's fine as it is, especially since it's `internal`, but `Comparator<String>` and extracting null comparators to the usages could add a little more transparency. Whatever you think is best